File: 125_Valid_Palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        strs = "".join(ch.lower() for ch in s if ch.isalnum())
        for i in range(len(strs)):
            if strs[i] != strs[(len(strs)-1-i)]:
                return False
        return True

File: cpp/125_Valid_Palindrome.cpp

class Solution {
public:
    bool isPalindrome(string s) {
        string str;
        for(char ch : s){
            ch = tolower(ch);
            if(isalnum(ch)){
                str += ch;
            }
        }
        string rev = str;
        reverse(rev.begin(),rev.end());
        if(str == rev ){
            return true;
        }
        else{
            return false;
        }
    }
};